lib/bundler/runtime.rb

```diff
@@ -193,6 +196,9 @@ def clean(dry_run = false)
       stale_gem_dirs       = gem_dirs - spec_gem_paths
       stale_gem_files      = gem_files - spec_cache_paths
       stale_gemspec_files  = gemspec_files - spec_gemspec_paths
+      stale_extension_dirs = extension_dirs - spec_extension_dirs
+
+      stale_extension_dirs.collect {|dir| FileUtils.rm_rf(dir) } unless dry_run
```

Comment:
Using `remove_dir` for this would duplicate the output of the removed gem, what should be the output text for this?